### ISSUE TITLE
Evaluation-only

### ISSUE THREAD

**Issue by charliebtan** [transferable-samplers/transferable-samplers#19]
It is currently not possible to only run evaluation for a given set of samples, it is required to initialize the model, data module etc.

It would be faster to avoid doing this.

We should also add a saving step for BG samples post-resampling (maybe also the "processed logits") so these can be handled without initalizing the model.

**Comment by charliebtan** [comment]
Another issue with CPU-only evaluation is the energy function can be impractically large for large numbers of systems.

A high-level workaround would be to save the energy evaluation with the samples.
